jupyter_elfinder/elfinder.py

```diff
@@ -420,21 +420,27 @@ def __places(self) -> None:
 
     def __open(self) -> None:
         """Open file or directory."""
-        if API_TARGET not in self._request:
+        init = self._request.get(API_INIT)
+        target = self._request.get(API_TARGET)
+        if not init and not target:
             self._response[R_ERROR] = "Invalid parameters"
             return
 
-        if API_INIT in self._request and self._request[API_INIT]:
+        if init:
             self._response[R_API] = 2.1
 
-        target = self._request.get(API_TARGET)
         if target:
             target = self.__find_dir(target)
             if not target:
-                self._response[R_ERROR] = "Invalid parameters"
+                if not init:
+                    self._response[R_ERROR] = "File not found"
+                    return
+                path = self._options["root"]
             elif not self.__is_allowed(target, "read"):
-                self._response[R_ERROR] = "Access denied"
-                return
+                if not init:
```

Comment:
I think this doesn't fit the spec: `If init == true and target is not set or that directory doesn't exist, then the data connector must return the root directory of the default volume. ` It didn't mention if the folder is not accessible. Maybe better to throw error for access denied?

Reply:
Why should we handle that error differently than if the target doesn't exist? I think it's more an oversight in the docs. I interpret the docs as if `init` is passed the backend should always set up regardless of target is ok or not.